.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# ide
+/.idea
```

Comment:
IntelliJ did produce `*.iml` files also in the past, are they gone in the newer versions?

Reply:
I'm using WebStorm right now and it only created the .idea directory